src/main/io/osd.c

```diff
@@ -687,6 +688,18 @@ static bool osdDrawSingleElement(uint8_t item)
             return true;
         }
 
+    case OSD_G_FORCE:
+        {
+            float osdGForce = 0;
+            for (int axis = 0; axis < 3; axis++) {
+                const float a = accAverage[axis];
+                osdGForce += a * a;
+            }
+            osdGForce = sqrtf(osdGForce) / acc.dev.acc_1G * 10;
+            tfp_sprintf(buff, "%01d.%01d%c", (uint8_t)osdGForce / 10, (uint8_t)osdGForce % 10, "G");
```

Comment:
No use of converting to `uint8_t`, parameters are passed as full words anyway

Reply:
Conversion is necessary, printf expects `int` as parameter (arguments are passed as variadic).

Compiler will handle promotion from `uint8_t` to `int` in variadic call

Reply:
BTW: `float` is always promoted to `double` in variadic arg, this was causing warning ...

Reply:
Only objection I had was about 8-bit type used rather than plain `int`

Reply:
I wouldn't call it variadic btw, it's a vararg (function with an ellipsis).
Variadic applies to variadic templates in C+-

Reply:
@DieHertz : Just using google for terminology ;-) https://www.gnu.org/software/libc/manual/html_node/Variadic-Functions.html

Reply:
I'm using ISO C and it has no word "variadic" in it, but it has "vararg" :-)